processing file asmFiles/aluop_and_alubranch.asm (1 of 36)
processing file asmFiles/beq_notTaken.asm (2 of 36)
processing file asmFiles/beq_taken.asm (3 of 36)
--- memsim.hex	2026-10-04 16:30:53.788696677 -0400
+++ memcpu.hex	2026-10-04 16:30:53.784696718 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :0400010000300213B6
 :040002000000029365
 :040003000050202366
processing file asmFiles/bge_notTaken.asm (4 of 36)
processing file asmFiles/bge_taken.asm (5 of 36)
--- memsim.hex	2026-10-04 16:30:56.471669242 -0400
+++ memcpu.hex	2026-10-04 16:30:56.466669293 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :0400010000100213D6
 :040002000000029365
 :040003000050202366
processing file asmFiles/bgeu_notTaken.asm (6 of 36)
processing file asmFiles/bgeu_taken.asm (7 of 36)
--- memsim.hex	2026-10-04 16:30:59.119642165 -0400
+++ memcpu.hex	2026-10-04 16:30:59.114642216 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :0400010000100213D6
 :040002000000029365
 :040003000050202366
processing file asmFiles/blt_notTaken.asm (8 of 36)
processing file asmFiles/blt_taken.asm (9 of 36)
--- memsim.hex	2026-10-04 16:31:01.686615916 -0400
+++ memcpu.hex	2026-10-04 16:31:01.681615967 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :0400010000300213B6
 :040002000000029365
 :040003000050202366
processing file asmFiles/bltu_notTaken.asm (10 of 36)
processing file asmFiles/bltu.taken.asm (11 of 36)
processing file asmFiles/aluop_and_alubranch.asm (1 of 36)
processing file asmFiles/beq_notTaken.asm (2 of 36)
processing file asmFiles/beq_taken.asm (3 of 36)
--- memsim.hex	2026-10-04 16:32:10.595911270 -0400
+++ memcpu.hex	2026-10-04 16:32:10.590911321 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :0400010000300213B6
 :040002000000029365
 :040003000050202366
processing file asmFiles/bge_notTaken.asm (4 of 36)
processing file asmFiles/bge_taken.asm (5 of 36)
processing file asmFiles/aluop_and_alubranch.asm (1 of 36)
processing file asmFiles/beq_notTaken.asm (2 of 36)
processing file asmFiles/beq_taken.asm (3 of 36)
--- memsim.hex	2026-10-04 16:38:42.788891347 -0400
+++ memcpu.hex	2026-10-04 16:38:42.783891399 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :0400010000300213B6
 :040002000000029365
 :040003000050202366
processing file asmFiles/bge_notTaken.asm (4 of 36)
processing file asmFiles/bge_taken.asm (5 of 36)
processing file asmFiles/beq_taken.asm (1 of 1)
--- memsim.hex	2026-10-04 16:40:42.766661525 -0400
+++ memcpu.hex	2026-10-04 16:40:42.762661567 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :0400010000300213B6
 :040002000000029365
 :040003000050202366
processing file asmFiles/beq_taken.asm (1 of 1)
processing file asmFiles/beq_taken.asm (1 of 1)
processing file asmFiles/beq_taken.asm (1 of 1)
--- memsim.hex	2026-10-04 17:05:31.626387111 -0400
+++ memcpu.hex	2026-10-04 16:54:25.346230478 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :0400010000300213B6
 :040002000000029365
 :040003000050202366
processing file asmFiles/beq_taken.asm (1 of 1)
processing file asmFiles/beq_taken.asm (1 of 1)
--- memsim.hex	2026-10-04 17:13:10.852661923 -0400
+++ memcpu.hex	2026-10-04 17:13:10.846661984 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :0400010000300213B6
 :040002000000029365
 :040003000050202366
